src/stdio/src/helpers.rs

```diff
@@ -0,0 +1,159 @@
+use super::{internal, BUFSIZ, FILE, UNGET};
+use compiler_builtins::mem::memset;
+use stdlib::calloc;
+use core::{mem, ptr};
+use core::sync::atomic::AtomicBool;
+use platform::types::*;
+use super::constants::*;
+use fcntl::*;
+use platform;
+use errno;
+
+/// Parse mode flags as a string and output a mode flags integer
+pub unsafe fn parse_mode_flags(mode_str: *const c_char) -> i32 {
```

Comment:
I don't see this function used anywhere. Is this a function you expect to be used in the future?

Reply:
`parse_mode_flags()` is used in `fdopen()` and `fopen()`